_topic_maps/_topic_map.yml

```diff
@@ -52,8 +52,10 @@ Name: Release notes
 Dir: release_notes
 Distros: openshift-enterprise
 Topics:
-- Name: OpenShift Container Platform 4.15 release notes
-  File: ocp-4-15-release-notes
+- Name: OpenShift Container Platform release notes
+  File: ocp-release-notes
+- Name: Other release notes
```

Comment:
For some reason, "Other" kind of bothers me. I think we could probably come up with a better option if we think about it long enough.

I'll think on it more, but my first other suggestions are:
"Additional release notes"
or
"Release notes reference"

:shrug:  Quick screenshots with these two options:

![image](https://github.com/user-attachments/assets/5a46ab71-cd87-4b4c-bf7c-27dd90dbce86)

---

![image](https://github.com/user-attachments/assets/167a9c8f-d499-4862-8c9e-2a88febb0a70)

Reply:
+1 for Additional

Reply:
Late on this comment, but I agree with "Additional".

Reply:
Same, agree with "Additional", but "Release notes reference" is pretty good too!